pkg/plugins/generic/generic_plugin.go

```diff
@@ -186,6 +186,11 @@ func needDrainNode(desired sriovnetworkv1.Interfaces, current sriovnetworkv1.Int
 		for _, iface := range desired {
 			if iface.PciAddress == ifaceStatus.PciAddress {
```

Comment:
No need to continue the loop after finding the device, maybe adding break at end of if

Reply:
fixed